changelog.md

```diff
@@ -16,8 +18,11 @@
   }
   deriving (Show, Generic)
 
+  deityTypeNameModifier True original = "Input" ++ original
+  deityTypeNameModifier False original = "Output" ++ original
```

Comment:
> e.g this schema will not fail. morpheus will generate types: `Deity` and `InputDeity`

hi, one last comment, since we say that it will generate `Deity` and `InputDeity` example implementation should match it.

we could write it like this (it's my preferred style, but old versions are good too):
```hs
deityTypeNameModifier :: Bool -> String -> String 
deityTypeNameModifier isInput original 
    | isInput = "Input" ++ original
    | otherwise = original
```

ps: you can mention yourself in the changelog if you like

Reply:
done!